src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageFormView.tsx

```diff
@@ -103,41 +108,56 @@ const CreateWorkPackageFormView: React.FC<CreateWorkPackageFormViewProps> = ({
     append: appendDeliverable,
     remove: removeDeliverable
   } = useFieldArray({ control, name: 'deliverables' });
-  const { fields: blockedBy, append: appendBlocker, remove: removeBlocker } = useFieldArray({ control, name: 'blockedBy' });
 
   const disableStartDate = (startDate: Date) => {
     return startDate.getDay() !== 1;
   };
 
   const { data: projects, isLoading, error, isError } = useAllProjects();
 
+  const {
+    data: project,
+    isLoading: isProjectLoading,
+    isError: useSingleProjectIsError,
+    error: useSingleProjectError
+  } = useSingleProject(validateWBS(wbsNum));
+
   if (isLoading || !projects) return <LoadingIndicator />;
 
   if (isError) {
     return <ErrorPage message={error?.message} />;
   }
 
+  if (useSingleProjectIsError) return <ErrorPage message={useSingleProjectError?.message} />;
+  if (!project || isProjectLoading) return <LoadingIndicator />;
+  const workPackages = project ? project.workPackages : [];
+
+  const blockedByOptions = workPackages.map(blockedByToAutocompleteOption);
+
   const blockedByFormControl = (
     <FormControl fullWidth>
       <FormLabel>Blocked By</FormLabel>
-      {blockedBy.map((_element, i) => {
-        return (
-          <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
-            <TextField required autoComplete="off" {...register(`blockedBy.${i}.wbsNum`)} sx={{ width: 9 / 10 }} />
-            <IconButton type="button" onClick={() => removeBlocker(i)} sx={{ mx: 1, my: 0 }}>
-              <DeleteIcon />
-            </IconButton>
-          </Grid>
-        );
-      })}
-      <Button
-        variant="contained"
-        color="success"
-        onClick={() => appendBlocker({ wbsNum: '' })}
-        sx={{ my: 2, width: 'max-content' }}
-      >
-        + ADD NEW BLOCKER
-      </Button>
+      <Controller
+        name="blockedBy"
+        control={control}
+        render={({ field: { onChange, value: formValue } }) => (
+          <Autocomplete
+            isOptionEqualToValue={(option, value) => option.id === value.id}
+            filterSelectedOptions
+            multiple
+            options={blockedByOptions}
+            getOptionLabel={(option) => option.label}
+            onChange={(_, values) => onChange(values.map((value) => value.id))}
+            value={formValue.map((value: string) => {
+              const change = blockedByOptions.find((option) => option.id === value);
```

Comment:
quick thing just delete the change thing to just be blockedByOptions.find...! and then delete the curly braces